projects/vdk-heartbeat/src/vdk/internal/heartbeat/job_controller.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, config: Config):
             f"with job {config.job_name} and team {config.job_team}. "
             f"Authorization endpoint: {auth_endpoint_message}"
         )
+        self.test_dict = {}
```

Comment:
What is the idea behind this variable ? Or is it forgotten debug var ...

Reply:
Good catch. It was indeed a test var. Removed, thanks